core/string_builder.h

```diff
@@ -70,6 +70,10 @@ class StringBuilder {
 		return appended_strings.size();
 	}
 
+	_FORCE_INLINE_ uint32_t get_string_length() const {
+		return string_length;
+	}
+
```

Comment:
@karroffel I just want to make sure this is ok

Reply:
Looks good!